benchmarks/dynamo/test.py

```diff
@@ -33,6 +33,7 @@ def test_benchmark_infra_runs(self) -> None:
                 [
                     "-dcpu",
                     "--inductor",
+                    "--training",
```

Comment:
@jansel looks like you added this file, does it need to be OSS or can i move it under fb folder so that i can make changes to it faster? The FB version of this PR D55389546 has more fixes